spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -308,58 +332,61 @@ function onChanOpenAck(
 function onChanOpenConfirm(
   portIdentifier: Identifier,
   channelIdentifier: Identifier) {
-  setActiveChannel(CounterPartyPortId)
+  // set the active channel for this owner/interchain account pair
+  setActiveChannel(portIdentifier)
 }
 ```
 
 ```typescript
 function onChanCloseInit(
   portIdentifier: Identifier,
-  channelIdentifier: Identifier) {
-    // users should not be able to close channels
-    return nil
+  channelIdentifier: Identifier) { \
+    // unset the active channel for this owner/interchain account pair
+    DeleteActiveChannel(portIdentifier, channelIdentifier)
```

Comment:
Shouldn't this return an error always? Ahh, is this because the ICA-auth may decide to close the channel? If so, then the line above about ICA channels only closing on timeouts should be changed

Reply:
This should return an error you are correct. I've updated this.